docs/src/pages/guides/new-notifications.md

```diff
@@ -0,0 +1,173 @@
+  url: string;
+}
+```
+
+The Whatsapp application however, requires a few more data to be passed to it. Below is an example.
+
+```javascript
+export interface WhatsappData extends MailData {
+  url: string
+  username: string
+  password: string
+}
+```
+
+## Linking
+
+After defining your interfaces and data types, we'll head over to `src/components/notification/index.ts` to link your notifications API. Find a function called `sendNotifications`, then add your app label you've defined above. This is where we also map the data from Monika, an array called `Notification[]` and an object called `NotificationMessage` which should be passed to your app.
+
+```typescript
+export async function sendNotifications(
+  notifications: Notification[],
+  message: NotificationMessage
+): Promise<void>
+```
+
+Going back to our example for google-chat, we linked `sendNotifications` to our API `sendGoogleChat()` like this:
+
+```typescript
+    case 'google-chat': {
+          await sendGoogleChat(notification.data, message)
+          break
+        }
+```
+
+## Implementation
+
+We now head over to `src/components/notification/channel/`. Create a new module specific for your application, something like `src/components/notification/channel/myChatApp.ts`. This is where all the notification implementations for each Monika event are coded. The events and messages you need to handle are embedded in the `NotificationMessage` object passed over from `sendNotifications()` above.
+
+Some of the events you need to handle are:
+
+- `start`: This is a "start-of-monitoring" message when monika is first fired up.
+- `termination`: A termination message occurs as Monika is shutting down.
+- `incident`: Incident is an alert triggered whenever we've detected an incident in the probes.
+- `recovery`: Recovery alert indicates the probe has recovered from the previous incident.
+- `status-update`: Status update is a daily summary message of the probe events.
+
+Back to our google-chat example, this is how we use `message.meta.type` to handle the different events.
+
+```typescript
+export const sendGoogleChat = async (
+  data: GoogleChatData,
+  message: NotificationMessage
+): Promise<any> => {
+
+  const notifType =
+    message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+
+  switch (message.meta.type) {
+    case 'start':
+      ... code to send start notifications
+    case 'termination':
+      ... code to send termination notifications
+    case `incident`:
+      ... code to send incident notifications
+
+```
+
+This is where you can get creative and really customize the look and feel of the notifications. You can use icons, widgets, fonts and colors of your choice. In the example `googlechat.ts` below, you can see how headers, text coloring are used to create the incident notifications. Your choices and available options will vary depending on your application. Please refer to your specific development documentation.
```

Comment:
Please refer to your specific development documentation. -> Refer to the app's development/integration documentation for more information.